examples/rust/votingmachine/proof/src/lib.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Risc0, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg_attr(not(test), no_std)]
+
+use r0vm_core::Digest;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct VotingMachineState {
+    pub polls_open: bool,
+    pub voter_bitfield: u32,
+    pub count: u32,
+}
+
+impl VotingMachineState {
+    pub fn check(&self) -> bool {
+        true
```

Comment:
Is this intended? What sort of things would we want to do in this function?

Reply:
Removed in latest commit 👍